instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/aws.go

```diff
@@ -60,6 +60,7 @@ func (m otelMiddlewares) initializeMiddlewareAfter(stack *middleware.Stack) erro
 		ctx context.Context, in middleware.InitializeInput, next middleware.InitializeHandler) (
 		out middleware.InitializeOutput, metadata middleware.Metadata, err error) {
 		serviceID := v2Middleware.GetServiceID(ctx)
+		operation := v2Middleware.GetOperationName(ctx)
```

Comment:
If this returns `""` should the span name fallback to just the service ID?

Reply:
Didn't find a semcov reference to the missing operation case - Currently, it will fallback to "ServiceID." I suggest removing the "." for this case or explicitly "ServiceID.NoOperation". WDYT?

Reply:
SGTM